stock_packaging_calculator/models/product_qty_by_packaging_mixin.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# @author: Sébastien Alix <sebastien.alix@camptocamp.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl)
+from odoo import api, fields, models
+
+
+class ProductQtyByPackagingMixin(models.AbstractModel):
+    """Allow displaying product qty by packaging.
+    """
+
+    _name = "product.qty_by_packaging.mixin"
```

Comment:
It does not look to me that the new mixin is covered by test, maybe I am mistaken ?

Reply:
no, I wanted to but I had no time to add fake models. BUT it's tested in #1168 on real models :wink:

Reply:
@TDu good for you? Then I can merge and rebase #1168